config/schema/elastic_apm.schema.yml

```diff
@@ -0,0 +1,48 @@
+elastic_apm.configuration:
+  type: config_object
+  label: 'Elastic APM Configuration'
+  mapping:
+    appName:
+      type: label
+      label: 'App name'
+    appVersion:
+      type: label
+      label: 'App version'
+    serverUrl:
+      type: label
+      label: 'Server URL'
+    secretToken:
+      type: label
+      label: 'Secret token'
+    hostname:
+      type: label
+      label: 'Host name'
+    timeout:
+      type: int
+      label: 'Guzzle Client timeout'
+    apmVersion:
+      type: label
+      label: 'APM version'
+    env:
+      type: sequence
+      label: '$_SERVER variables'
+      sequence:
+        type: string
+    cookies:
+      type: sequence
+      label: 'Cookies'
+      sequence:
+        type: string
+    httpClient:
+      verify:
+        type: boolean
+        label: 'Verify'
+      proxy:
+        type: label
+        label: 'Proxy'
+    capture_exceptions:
+      type: boolean
+      label: 'Capture errors and exceptions'
+    active:
+      type: boolean
+      label: 'Enabled'
```

Comment:
The reason I was using camelCase was because the config names used in the actual Elastic APM agent are camelCase as well. So, saving the configuration in Drupal that way just makes it easier for me, where all I have to do is just take the config and send to the Agent w/o having to convert the labels.

Reply:
Ok, let's find a way to keep this both convenient but also structured. Let's merge it as is for now so that we can move on with the convention to follow `camelCase` and we will review the config structure before making an alpha release. It's also wrong to pass the full configuration array to the agent's constructor, we should be passing only the configuration needed - I've commented on the relevant piece of code in PR 3. We'll fix these later.